src/trait_data.R

```diff
@@ -0,0 +1,12 @@
+library(traits)
+options(betydb_key = readLines('~/.betykey', warn = FALSE),
+        betydb_url = "https://terraref.ncsa.illinois.edu/bety/",
+        betydb_api_version = 'beta')
+
+system.time(
+canopy_cover <- betydb_query(table = 'search',
+                               trait = "canopy_cover",
+                               site  = "~Season 6",
+                               limit = '100')
+)
```

Comment:
don't need system.time?

Reply:
Is this already living somewhere in R or do we have to approve it first? What exactly does this snippet do?

Reply:
you approve and merge it into your repository so that you can later modify the code to import more records and additional data